app/services/course_search_service.rb

```diff
@@ -76,6 +83,12 @@ def call
 
   private
 
+  def course_type
+    return :undergraduate if @course_type_answer_determiner.show_undergraduate_courses?
+
+    :postgraduate
```

Comment:
Although for the web this will work, for the API it won't. I will create a separate card to deal with API since in 2025 we want to show all courses when using this service class.